src/frontend/src/lib/components/hero/Hero.svelte

```diff
@@ -1,12 +1,16 @@
 <script lang="ts">
+	import { slide } from 'svelte/transition';
 	import HeroContent from '$lib/components/hero/HeroContent.svelte';
+	import { SLIDE_PARAMS } from '$lib/constants/transition.constants';
 
 	export let usdTotal = false;
 	export let summary = false;
-	export let actions = true;
 	export let back = false;
 </script>
 
-<article class="main relative flex flex-col items-center rounded-lg pb-6 pt-10">
-	<HeroContent {usdTotal} {summary} {actions} {back} />
+<article
+	class="relative flex flex-col items-center rounded-lg pb-6"
+	transition:slide={SLIDE_PARAMS}
```

Comment:
There is no `#if`, this transition has no effect no?

Reply:
The IF condition is outside the component (see `+layout` page), so this one works

Reply:
Really? If was under the impression that Svelte requires an animation to be preceded by a IF. Good to know.

Reply:
Looking at [documentation](https://svelte.dev/docs/element-directives#transition-fn), it actually mention it, I didn't know before testing

> A transition is triggered by an element entering or leaving the DOM as a result of a state change.
> 
> When a block is transitioning out, all elements inside the block, including those that do not have their own transitions, are kept in the DOM until every transition in the block has been completed.